contrib/benitlux/.gitignore

```diff
@@ -1,3 +1,4 @@
 src/benitlux_serial.nit
 *.db
 *.email
+benitlux_corrections.txt
```

Comment:
I though about it but it's the kind of file that I will update often. I prefer for it to be ignored by git so I can keep it up to date and switch branches at the same time. Also, these are historical errors, they go with the content of the DB.